go/service/kbfs_mount.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package service
+
+import (
+	"golang.org/x/net/context"
+
+	"github.com/keybase/client/go/libkb"
+	//	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-framed-msgpack-rpc/rpc"
+)
+
+type KBFSMountHandler struct {
+	*BaseHandler
+	libkb.Contextified
+}
+
+func NewKBFSMountHandler(xp rpc.Transporter, g *libkb.GlobalContext) *KBFSMountHandler {
+	return &KBFSMountHandler{
+		BaseHandler:  NewBaseHandler(xp),
+		Contextified: libkb.NewContextified(g),
+	}
+}
+
+func (h *KBFSMountHandler) GetCurrentDriveLetter(ctx context.Context) (res string, err error) {
```

Comment:
This behavior is a little bit confusing given the name of the RPC.  I wouldn't expect this RPC to set the drive letter to the next available default drive on the basis of the RPC's name.  If this is actually the behavior you want, can you change the name of the RPC to be more descriptive? The correct RPC name is pretty crazy:

`GetCurrentDriveLetterOrSetToNextAvailable`

Maybe the right idea is to move this logic over to the client (in this case KBFS), and just expose `Get...`, `GetAllAvailable...` and `Set`.